virttest/utils_test/libvirt.py

```diff
@@ -3550,3 +3550,25 @@ def check_qemu_cmd_line(content, err_ignore=False):
             raise exceptions.TestFail("Expected '%s' was not found in "
                                       "qemu command line" % content)
     return True
+
+
+def get_disk_alias(vm, source_file=None):
+    """
+    Get alias name of disk with given source file
+
+    :param vm: VM object
+    :param source_file: domain disk source file
+    """
+    if vm.is_alive():
+        ori_vmxml = vm_xml.VMXML.new_from_dumpxml(vm.name)
+        disks = ori_vmxml.devices.by_device_tag('disk')
+        for disk in disks:
+            if disk.xmltreefile.find('source') is not None:
+                if disk.source.attrs['file'] == source_file:
```

Comment:
merge to line 3567

Reply:
if disk.xmltreefile.find('source') and disk.source.attrs['file'] == source_file:

Reply:
I need "else" for no "source" condition so can not merge them